### PR TITLE
Fix compile-time errors

### PR DESCRIPTION
**Pull request by kerkovits**
Discussed in https://github.com/GrandOrgue/grandorgue/discussions/1434

A separate commit corrects two minor typos in BUILD.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Fixed building with gcc13. https://github.com/GrandOrgue/grandorgue/discussions/1434
```
[the remark this answers is not in the document]
Comment (kerkovits) [reply CHANGELOG.md]: Are there any formal rules exactly which changes should we include in the change log?

**Comment by larspalo** [comment]
If we're moving towards a more modern c++ approach generally in the code base, shouldn't we use <cstdint> and fully qualify each variable with std:: instead of using the depracated <stdint.h>?

**Comment by kerkovits** [comment]
I would leave it for @oleg68's discretion. There are a handful of files already including stdint.h, and even more including it implicitly via other headers. There is no example in the present codebase for std:: integer types. My intention was to adhere present coding style, while make it possible to build GrandOrgue in the beta version of Fedora. If the decision is to replace `<stdint.h>` with `<cstdint>`,  that should be a separate pull request (some clever regex replace could simply solve it), as it would affect significantly more files.

**Comment by larspalo** [comment]
@kerkovits I would not leave it up to just @oleg68 to single-handedly decide upon what standard GrandOrgue coding should adhere to. Many of his recent PRs show a significant move to more recent c++ coding approaches, and to not work against the overall scheme, all new code additions should rather move in the same direction or not be included at all.

**Comment by oleg68** [comment]
I agree with @larspalo that `<cstdint>` is better than `<stdint.h>`.
I agree with @kerkovits that replacing `<stdint.h>` with `<cstdint>` in all existing source files should be a separate PR.

So I suggest to **add** `<cstdint>`  where there were neither `<cstdint>` nor `<stdint.h>` before.

And you are welcome to open a separate PR for replacing `<stdint.h>` with `<cstdint>` in other source files.

**Comment by larspalo** [comment]
I agree completely with @oleg68. Just remember to also qualify with std:: before the type in the files where <cstdint> are added.

**Comment by oleg68** [comment]
@larspalo `<cstdint>` does not require using the `std::` prefix.

**Comment by larspalo** [comment]
> @larspalo `<cstdint>` does not require using the `std::` prefix.

Well, it "can" work without, because the implementation that are used "might" put the required symbols in the global namespace, but it's not guaranteed (they really should be in namespace std). The right thing to do is to qualify the types with std:: to be sure that it will work.

**Comment by kerkovits** [comment]
Exactly this was the reason why I used `<stdint.h>` instead of `<cstdint>`: it is almost impossible to follow, which other headers include the headers affected in this PR: those headers would also need the `std::` namespace declaration. I suggest to:
- Accept this PR as is.
- Use `sed` to change all occurrences of `<stdint.h>` to `<cstdint>` and `/u?int(8|16|32|64)_t/` to `std::\0`. The code-base should build after this regex replace. If not, debug. Finally, do a PR.

**Comment by oleg68** [comment]
> I used `<stdint.h>` instead of `<cstdint>`: it is almost impossible to follow, which other headers include the headers affected in this PR: those headers would also need the `std::` namespace declaration

Have you tried to use `<cstdint>` only in the five new .h files without `std::`? If you have and it works, I'd suggest to do this in this PR. This change seems to be minimal in the right direction.

**Comment by kerkovits** [comment]
I am trying it now whether it builds.

**Comment by kerkovits** [comment]
Results will be here: https://copr.fedorainfracloud.org/coprs/kerkovits/GOtest/build/5695226/

**Review by oleg68 — APPROVED** [review]

**Review by larspalo — APPROVED** [review]

**Pull request merged**